searches/jump_search.py

```diff
@@ -1,7 +1,28 @@
+"""
+Pure Python implementation of the jump search algorithm.
+This algorithm iterates through a sorted collection with a step of n^(1/2),
+until the element compared is bigger than the one searched.
+It will then perform a linear search until it matches the wanted number.
+If not found, it returns -1.
+"""
+
 import math
 
 
-def jump_search(arr, x):
+def jump_search(arr: list, x: int) -> int:
+    """
+    Pure Python implementation of the jump search algorithm.
+    Examples:
+    >>> jump_search([0, 1, 2, 3, 4, 5], 3)
+    3
+    >>> jump_search([-5, -2, -1], -1)
+    2
+    >>> jump_search([0, 5, 10, 20], 8)
+    -1
```

Comment:
I tend to like to raise an exception like str.index() vs str.find() but that is a personal choice.